awesomeConfig.cmake

```diff
@@ -67,6 +67,12 @@ a_find_program(CONVERT_EXECUTABLE convert TRUE)
 include(FindPkgConfig)
 # lua
 include(FindLua)
+if (NOT LUA_FOUND)
+    message(FATAL_ERROR
+        "Could not find Lua. See the error above.\n"
+        "You might want to hint it using the LUA_DIR environment variable, "
+        "or set the LUA_INCLUDE_DIR / LUA_LIBRARY CMake variables.")
+endif()
```

Comment:
If so, shouldn't that be done in CMake instead of us working around CMake? (I'm not sure how well it fits into this PackageHandleStandardArgs-macro, however...).
Also, I don't think the hint to `LUA_DIR` works. If I read `FindLua.cmake` correctly, it looks for `${subdir}/lua.h` where `${subdir}` is `include/luaXY`, `include/luaX.Y` and `include/lua-X.Y`, and `X` is always 5 and `Y` ranges from 0 to 3. So, `LUA_DIR` is useless...?

Thus, this would only help users who have a non-system (or non-package-manager?) installation of Lua. That's basically no one, is it?

The more common error is version mismatches (5.2 headers, but 5.3 library, or something like that) and we cannot do much about that.